.travis.yml

```diff
@@ -2,7 +2,8 @@ language: scala
 jdk:
   - oraclejdk8
```

Comment:
Looks like you can:

script:
  - jdk_switcher use oraclejdk8
  - # do stuff with Java 8
  - jdk_switcher use oraclejdk7
  - # do stuff with Java 7

http://docs.travis-ci.com/user/languages/java/